### ISSUE TITLE
Piclists not correctly named/placed/saved in piclists folders

### ISSUE THREAD

**Issue by KimB2017** [sillsdev/ptx2pdf#242]
In 1.2.9
I don't see tickbox for "Use piclist". If there is no piclist for a book, illustrations are ignored. It looks as if ptxprint has generated a blank piclist for each of my test-no-piclist runs. Interestingly, these blank autogenerated piclist files are the only piclists that correctly list the BOOK code in the file name. See annotated screenshot for what I think are oddities/bugs with names and folder placement of piclist files. 

Even if you've dropped the use of "piclist" folders within the shared/ptxprint tree, when I click to generate a piclist it is not readily identifiable because the MAT-book code is not included. The only piclist files which include book code are the blank, autogenerated ones that appear when I have not requested them (trying to test printing pics without a piclist). These blank files cause all pics to be omitted from a Print run.
![Selection_014](https://user-images.githubusercontent.com/25071218/96492055-0c988c00-1211-11eb-987a-a68f4abf578f.png)

**Comment by KimB2017** [comment]
So then, after closing/reopening ptxprint I can't get to "RTTT-None.piclist" at all, it seems, because None is not available as a configuration name on the Basic tab. But this list is my good, current piclist with my MAT edits and I want it back.

Nor do I want to lose this MAT piclist. When I chose the MattTest1-2-9 config it showed my XXA piclist. But this was overwritten when I changed the book to ACT. 

I do need to be able to save my edited piclist for each book; it takes a lot of work to tweak them. And it would be very helpful if I could retrieve a Default-MAT... list, for example, and reuse it in a different Saved Config (which corresponds to a printed book?? or to a layout that I might want to reuse for multiple books, in contrast to a different type of printed book (size, audience, etc).

**Comment by KimB2017** [comment]
So then I created a new Saved Configuration, and tested the new Module option on Basics tab. For the first time in months I was able to print my ChristmasStory module with resized illustrations! So I did some editing (in gedit, outside ptxprint) of the piclist file that was generated (at Paratext8Projects/RTTT/shared/ptxprint/ChristmasModule1-2-9/RTTT-ChristmasModule1-2-9.piclist). I saved this file in gedit, then returned to ptxprint and clicked to "Refresh" the piclist on the View+Edit tab. But nothing happened. When I closed ptxprint I saw that my manually edited file (what I wanted to keep) was overwritten by the pre-edited version that ptxprint was displaying.

However, when I reopen ptxprint I can not access this piclist, even though I invoked the ChristmasModule1-2-9 Saved Configuration.

**Comment by KimB2017** [comment]
I think maybe ptxprint is overwriting the "correct" copyright statements I am adding to a piclist via the Edit Details tab, and keeping what is in the Paratext file. (Someone else made a similar observation last week or so.) But what is in the PT file does not appear in ptxprint anywhere; I can only see it by viewing a piclist file externally.

**Comment by davidg-sil** [comment]
I'm pretty sure that rather than deleting piclists, they were going to get renamed to whatever.piclist.old. At least that's what MH said... But it seems some other fixes from roughly that time have confusingly evaporated so maybe an accident happened here too. I hope not.

**Comment by davidc86** [comment]
I think v1.3 now fixes a lot of these issues. At least it seems so on my machine.

**Comment by KimB2017** [comment]
I'm still having problems with piclist behavior in 1.3: 
I opened ptxprint and saved my settings to "Mattest". Then I edited a few fig specs, and saved the Mattest config again. Then I closed and reopened ptxprint (back to Mattest). None of my piclist edits were saved! 

What I see in my PT project folder now is **three** piclists: 
(1)  RTTT/PrintDraft/**ptxprint-Mattest-MATRTTT**.piclist, which has my edits, 
(2)  RTTT/shared/ptxprint/**RTTT-None**.piclist and 
(3)  RTTT/shared/ptxprint/Matttest/**RTTT-Mattest**.piclist; both of the latter seem to have been created automatically and have none of my tweaks, nor do they have the (MAT) book id in their filenames. 

So I saved a new Config "Newtest". It added a new file parallel to (3) above: shared ptxprint/Newtest/RTTT-Newtest.piclist,
and one parallel to (1): PrintDraft/ptxprint-Newtest-MATRTTT.piclist. After close/reopen I can't get to this one, which has my edits. 

It seems that perhaps the goal should be to save the (1) version in the shared/[config]/ folder  and then reload it the next time. But it instead gets saved to the project's PrintDraft folder and a new default piclist gets created in shared/[config]/.

So how can I get ptxprint to load ptxprint-Mattest-MATRTTT.piclist, rather than loading a new default piclist when I open ptxprint? Seems like this version, including the book id, should be saved in RTTT/shared/ptxprint/
